resources/dicts/conditions/pregnancy.json

```
@@ -1,6 +1,9 @@
 {
   "announcement": [
-    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits."
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is when m_c announced {PRONOUN/m_c/poss} pregnancy!",
```

```suggestion
	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today when m_c announced {PRONOUN/m_c/poss} pregnancy!",
```

---

```suggestion
	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is, when m_c announced {PRONOUN/m_c/poss} pregnancy!",
```
both are possible, depends what you wanna say with it